modules/learn/pages/clusters-and-availability/xdcr-conflict-resolution.adoc

```diff
@@ -117,3 +117,34 @@ Therefore, in this instance, timestamp-based conflict resolution should be used,
 XDCR replications cannot be created between buckets with different conflict resolution policies: source and target buckets must always be configured with the same policy.
 
 When using XDCR with a source cluster running a pre-4.6 version of Couchbase Server, only conflict resolution based on _sequence numbers_ can be used.
+
+[#monitoring-conflict-resolution]
+== Monitoring Conflict Resolution
+
+Conflict resolution can be _monitored_, by means of statistics provided for the REST API and _Prometheus_.
+The statistics cover three scenarios: for each, two statistics are provided; corresponding to attempts respectively to modify and to delete a local document through conflict resolution.
```

Comment:
The statistics cover three scenarios: for each, two statistics are provided; corresponding to attempts respectively to modify and to delete a local document through conflict resolution.  Please note that the statistics only apply to documents that are sent to the target cluster by the XDCR running on the source.  XDCR conflict resolution can be done on the source, as well as on the target, and in those cases, where the source decides not to send a document to the target, the kv_conflicts_resolved statistics on the target would not capture those scenarios.  Please note that there are also xdcr_docs_failed_cr_source_total and xdcr_docs_failed_cr_target_total metrics, also available via REST API and _Prometheus_.